doc/api/tls.md

```diff
@@ -650,6 +650,10 @@ Methods that return TLS connection metadata (e.g.
 [`tls.TLSSocket.getPeerCertificate()`][] will only return data while the
 connection is open.
 
+The `highWaterMark` parameter of the duplex [Stream][] makes no sense for
+`tls.TLSSocket`, because the TLS protocol assumes 16KB as the maximum record
+size.
```

Comment:
I suggest to move this comment down below and to mention that it's not supported by extending the following entry ```* ...: Any [`socket.connect()`][] option not already listed.```.

Reply:
Moved it. Please check the update.